webapp/tests/mocha/unit/views/doc_summaries_by_id.spec.js

```diff
@@ -96,7 +96,8 @@ const postNatalVisit = {
 const postNatalVisitBis = Object.assign({}, postNatalVisit, {
   _id: '4971a859-bde7-4ff0-a0ed-326925b83038-bis',
   fields: Object.assign({}, postNatalVisit.fields, {
-    patient_id: null
+    patient_id: null,
```

Comment:
Please add a couple more tests here where: 

- we have patient_id and no patient_uuid
- we don't have patient_id, but we have patient_uuid
- we have both patient_id and patient_uuid

Reply:
Got it ! add, add the first two, I think the last one is already covered.